app/grants/models.py

```diff
@@ -364,6 +364,9 @@ class Meta:
 
     # Grant Query Set used as manager.
     objects = GrantQuerySet.as_manager()
+    verified = models.BooleanField(default=False)
+    verified_by = models.ForeignKey('dashboard.Profile', null=True, blank=True, on_delete=models.SET_NULL)
+    verified_at = models.DateTimeField(blank=True, null=True)
```

Comment:
^ Is this meant only for twitter ! If so rename all by adding prefix `twitter`
Also also help_text attribute explaining what it is (even if it's obvious)

Reply:
done